apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/rivers/JournalfoerVedtaksbrev.kt

```diff
@@ -31,7 +30,7 @@ internal class JournalfoerVedtaksbrev(
 
     init {
         River(rapidsConnection).apply {
-            eventName(KafkaHendelseType.ATTESTERT.toString())
+            eventName(BrevEventTypes.FERDIGSTILT.toString())
```

Comment:
Er dette trygt å endre for den opprinnelige flyten til denne riveren også?
Er ikke så godt kjent med flyten her lengre, men så ikke umiddelbart en tilsvarende endring noe annet sted.

Reply:
Eg vart litt overraska her sjølv. Tanken er at denne riveren hektar seg inn i flyten før `JournalfoerVedtaksbrev` overalt, og så sender vi for alt anna berre direkte vidare. Så derfor sender `OpprettVedtaksbrev`-riveren ut `BrevEventTypes.FERDIGSTILT` som denne skal plukke opp.